client/sites-dashboard/components/sites-tile-item.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from '@emotion/css';
+import { useI18n } from '@wordpress/react-i18n';
+import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+import { displaySiteUrl, getDashboardUrl } from '../utils';
+import { SiteStatusBadge } from './site-status-badge';
+import { SitesEllipsisMenu } from './sites-ellipsis-menu';
+import SitesLaunchStatusBadge from './sites-launch-status-badge';
+import SitesP2Badge from './sites-p2-badge';
+import { SiteName } from './sites-site-name';
+import { SiteUrl } from './sites-site-url';
+
+const container = css( {
+	display: 'flex',
+	width: '100%',
+	flexDirection: 'column',
+	minWidth: 0,
+} );
+
+const siteTitle = css( {
+	display: 'flex',
+	flex: 1,
+	marginTop: '16px',
+	marginBottom: '8px',
+	alignItems: 'center',
+} );
+
+const badges = css( { display: 'flex', gap: '8px', alignItems: 'center', marginLeft: 'auto' } );
+
+const siteImage = css( {
+	aspectRatio: '1 / 1',
+	width: '100%',
+	background: '#F6F7F7',
+} );
+
+const ellipsis = css( {
+	'.button': {
+		padding: '0 !important',
+	},
+
+	'.gridicon': {
+		width: '24px !important',
+		height: '16px !important',
+		top: '4px !important',
+	},
```

Comment:
Yikes...

Reply:
Ya! I think many of our shared components were just always assuming styles would be overridden using CSS of a higher precedence so this sort of thing never came up. Like the rule would always have multiple class names like

`.sitesTileItem .ellipsis.gridicon {`

Which means the `!important` was never needed. I guess it's an issue of trying to interface Emotion components with non-Emotion.

Reply:
Yep, and this sucks. Let me know if there's a better option, I can't think of anything on top of my head that doesn't include refactoring the `EllipsisMenu` component.